docs-website/sidebars.js

```diff
@@ -106,6 +106,15 @@ module.exports = {
         ],
       },
     ],
+    "Integration Guides": [
+      {
+        "BigQuery": [
+          "docs/quick-ingestion-guides/Page_1-Overview",
```

Comment:
By default this is going to show the H1 header from the doc, for example `Page_1-Overview.md` will show up on the sidebar as `DataHub BigQuery UI Ingestion Guide: Overview`

My suggestion would be to have the following nav experience:
```
Integration Guides
|_ BigQuery
   |- Overview
   |- Setup
   |_ Configuration
````

We have a couple of options:
1. Hardcode sidebar value mappings here: https://github.com/datahub-project/datahub/blob/master/docs-website/generateDocsDir.ts#L174-L180
2. Remove specific prefixes from H1 here: https://github.com/datahub-project/datahub/blob/master/docs-website/generateDocsDir.ts#L246-L255 

Details are [here](https://github.com/datahub-project/datahub/tree/master/docs-website#sidebar-display-options)